### PR TITLE
Movie list/tile element

### PR DESCRIPTION
**Pull request by damdlani**

### CODE REVIEW

File: src/components/Tile/styled.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
+import styled, { css } from 'styled-components';
+
+export const TileElement = styled.div`
+    max-width: 1368px;
+    margin: 100px auto;
```
Comment (anetaszynal) [inline src/components/Tile/styled.js:5]: Isn't it 56 px? (150-94)

File: src/components/Tile/styled.js [hunk 2]
```diff
@@ -0,0 +1,162 @@
+import styled, { css } from 'styled-components';
+
+export const TileElement = styled.div`
+    max-width: 1368px;
```
Comment (anetaszynal) [inline src/components/Tile/styled.js:4]: I think it is unnecessary here.

File: src/components/Tile/styled.js [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+    margin: 100px auto;
+    padding: 0px 0px 0px 40px;
+    background-color: ${({theme}) => theme.color.white};
+    box-shadow: 0px 4px 12px rgba(186, 199, 213, 0.5);
```
Comment (anetaszynal) [inline src/components/Tile/styled.js:8]: border-radius is missed

File: src/components/Tile/styled.js [hunk 4]
```diff
@@ -0,0 +1,162 @@
+import styled, { css } from 'styled-components';
+
+export const TileElement = styled.div`
+    max-width: 1368px;
+    margin: 100px auto;
+    padding: 0px 0px 0px 40px;
```
Comment (anetaszynal) [inline src/components/Tile/styled.js:6]: Could you tell me where these values ​​come from? I can't see them. Wouldn't the grid be a better solution here and the grid-gap?

File: src/components/Tile/styled.js [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+`;
+export const AdditionalInfo= styled.span`
+    margin-right: 10px;
+    font-size: 18px;
+    font-weight: 400;
+    line-height: 1.2;
+    color: ${({theme}) => theme.color.stormGray};
+    ${({content}) => content && css`
+        color: ${({theme}) => theme.color.black};
+        font-weight: 500;
```
Comment (anetaszynal) [inline src/components/Tile/styled.js:96]: In movie details font weight is 400

File: src/components/Tile/styled.js [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+        margin: 0 12px 0 0;
+    `}
+`;
+export const Rate = styled.div`
+     color: ${({theme}) => theme.color.black};
+     font-size: 14px;
+     margin: 0 8px 0 0;
+
+     ${({number}) => number && css`
+        font-size: 22px;
```
Comment (anetaszynal) [inline src/components/Tile/styled.js:138]: line-heigh is missing and position is diffrent.

File: src/components/Tile/styled.js [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+    flex-direction: column; 
+
+    ${({tileView}) => tileView === "list" && css`
+        margin: 0;
+        padding: 16px;
+        width: 100%;
+        height: 100%;
+    `}
+`;
+export const Header = styled.h1`
```
Comment (anetaszynal) [inline src/components/Tile/styled.js:44]: In my opinion it should be h2.

**Review by anetaszynal — CHANGES_REQUESTED** [review]
Good job! 😉 Please consider using grid.

File: src/components/Tile/styled.js (hunk 4 above)
Comment (damdlani) [reply src/components/Tile/styled.js:6]: I forgot to delete it. I used it just to better see the element during creating. Grid should be used in Michał's tile container.

File: src/components/Tile/styled.js (hunk 7 above)
Comment (damdlani) [reply src/components/Tile/styled.js:44]: I wondered about it. But where is h1 used then? If nowhere, I can't see a better place for it

Comment (damdlani) [reply src/components/Tile/styled.js:44]: Right, h1 should be used for container title, not movie title

File: src/components/Tile/styled.js (hunk 5 above)
Comment (damdlani) [reply src/components/Tile/styled.js:96]: Changed it ;)

File: src/components/Tile/styled.js (hunk 6 above)
Comment (damdlani) [reply src/components/Tile/styled.js:138]: Changed it ;)

**Review by anetaszynal — APPROVED** [review]

**Pull request merged**